modules/tinymce/src/core/main/ts/api/html/Sanitization.ts

```diff
@@ -152,6 +157,25 @@ const setupPurify = (settings: DomParserSettings, schema: Schema): DOMPurifyI =>
     }
   });
 
+  purify.addHook('afterSanitizeAttributes', (node) => {
+    const { attributes } = node;
+    const currentOrder = Arr.from(attributes);
```

Comment:
Why did you copy the attributes to an array, and then search it?

There's no need, you can just do an object lookup as the original suggestion did.

Reply:
changed